contracts/icns-registrar/src/contract.rs

```diff
@@ -239,6 +240,8 @@ pub fn execute_claim(
     check_verfying_msg(deps.as_ref(), &env, &info, &name, &verifying_msg_str)?;
     let is_admin = is_admin(deps.as_ref(), &info.sender)?;
     if !is_admin {
+        // NL: Question: Where is the backend that accumulates these signatures
+        // and generates the claim message? or is it the FE?
```

Comment:
the claim msg is generated from FE, send to verifiers and verifiers send back signature, FE accumulates them and send tx.

Verifier code is here, asked richard to open source it https://github.com/icns-xyz/icns-verifier